src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/info/network/Path.java

```diff
@@ -37,11 +37,16 @@ public class Path implements Serializable {
     @JsonProperty("nodes")
     private List<PathNodePayload> nodes;
 
+    @JsonProperty("isBackupPath")
```

Comment:
Please use snake_case for json property naming.

Reply:
Fixed